.env.example

```diff
@@ -5,6 +5,9 @@ NETWORK=ganache
 # DOES NOT NEED TO BE SET WHEN DEVELOPING LOCALLY
 NETWORK_CONTRACT_ADDRESS=0x0000000000000000000000000000000000000000
 
+HOST="localhost:9091"
+ORIGIN="http://localhost:9091"
```

Comment:
I don't think that the origin is actually needed. I think we can just get away with using just the host.

The communication protocol should still come from the client, `window.location.protocol`, to which we just append the host